ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1039,3 +1039,44 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+
+      - '{name}-{test_name}-for-pull-request':
+          test_name: multicluster-e2e
+          node: 'antrea-multicluster-node'
+          description: 'This is the {test_name} test for {name} pull request.'
+          white_list_target_branches: [ ]
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
+          status_context: jenkins-multicluster
+          status_url: --none--
```

Comment:
I think we don't need to set ` status_url: --none--`  for jobs in projects-cloud.yaml. This is because the jobs are updated to a Jenkins on public cloud, and the testbed (Jenkins node) is also on public cloud, so we should provide the job result URL. Setting ` status_url: --none--`  is only for private Jenkins.

Reply:
Changed `--none--` to the job's URL.